apps/web-giddh/src/app/invoice/proforma/proforma-list.component.html

```diff
@@ -92,7 +73,7 @@
         </div>
     </div>
 
-    <div class="mobileSearch mrB1">
+    <div class="mobile-search mrB1">
```

Comment:
change mrB1 class name

Reply:
as we can see dark green is added on this class, i agree mrB1 should have been changed but it's ignored ? is it need to update ?